common/tools/field_test_log_plotter/ftl_plotter.py

```diff
@@ -400,6 +400,17 @@ def __convert_units(self):
             self.df['RX throughput [Bits/s]'] = self.df['RX throughput [Bits/s]'] / tp_divider
             self.df['TX throughput [Bits/s]'] = self.df['TX throughput [Bits/s]'] / tp_divider
 
+        tp_divider = 1
+        # Convert throughput values
+        if self.throughput_units == 'Kb':
+            tp_divider = 1000
+        elif self.throughput_units == 'Mb':
+            tp_divider = 1000000
+        if tp_divider > 1:
+            # Converting only actual values but not changing column name.
+            self.df['RX throughput [Bits/s]'] = self.df['RX throughput [Bits/s]'] / tp_divider
+            self.df['TX throughput [Bits/s]'] = self.df['TX throughput [Bits/s]'] / tp_divider
+
```

Comment:
Why to add second time same code block?

Reply:
You are right, this is merge glitch. Duplicate code removed.